### PR TITLE
[ADD] hr_timesheet_validator

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This is a proposal to add module 'hr_timesheet_validator'. 

This PR substitutes PR 19.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
Substitutes PR https://github.com/OCA/hr-timesheet/pull/19

**Comment by mdietrichc2c** [comment]
Still 2 PEP8 issues on this PR:

`./hr_timesheet_validator/model/hr_timesheet_sheet.py:31:80: E501 line too long (80 > 79 characters)`

`./hr_timesheet_validator/model/hr_timesheet_sheet.py:106:80: E501 line too long (80 > 79 characters)`

File: hr_timesheet_validator/model/hr_timesheet_sheet.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                        cr, uid, res_id)
+                    group_hr_user_ids = [user.id for user
+                                         in group_hr_user.users]
+                    if uid in group_hr_user_ids:
+                        continue
+
+            validator_user_ids = []
+            for validator_user_id in timesheet.validator_user_ids:
+                validator_user_ids.append(validator_user_id.id)
+            if uid not in validator_user_ids:
```
Comment (mdietrichc2c) [inline hr_timesheet_validator/model/hr_timesheet_sheet.py:130]: Definitely a case for list comprehension: `validator_user_ids = [x.id from x in timesheet.validator_user_ids]`

File: hr_timesheet_validator/model/hr_timesheet_sheet.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+from openerp import netsvc
+
+
+class hr_timesheet_sheet(orm.Model):
+    _inherit = "hr_timesheet_sheet.sheet"
+
+    _columns = {
+        'validator_user_ids': fields.many2many('res.users',
+                                               string='Validators',
+                                               required=False),
```
Comment (mdietrichc2c) [inline hr_timesheet_validator/model/hr_timesheet_sheet.py:33]: required is `False` by default

**Comment by feketemihai** [comment]
@jbeficent I will close this one since long time without activity, if still need it please reopen the PR.

**Pull request closed without merging**